pkg/csr/csr.go

```diff
@@ -91,9 +91,9 @@ func (a *Approver) Approve() error {
 	approvalCond := certificates.CertificateSigningRequestCondition{
 		Type:           certificates.CertificateApproved,
 		Status:         "True",
-		Message:        "Approved by WMCO",
+		Message:        "This CSR was approved by the WMCO certificate Approver.",
 		LastUpdateTime: meta.Now(),
-		Reason:         "WMCOApproved",
+		Reason:         "WMCOApprove",
```

Comment:
The test would need to be updated as well to look for this reason in the approved CSR's